app/views/home/index.html.erb

```diff
@@ -0,0 +1,61 @@
+<section>
+  <div class="card">
+    <div class="card-body">
+      <h2><span><strong>Media Spotlight: </strong></span><span class="list-titles"><%= link_to Work.top_vote_getter.title, work_path(Work.top_vote_getter.id) %></span><span class="list-titles"> by <%= Work.top_vote_getter.creator %> </span></h2>
+      <p>
+        <%= Work.top_vote_getter_votes %> votes - <%= Work.top_vote_getter.description %>
+      </p>
```

Comment:
Rather than having the views call model methods directly, these method should be invoked in the controller and stored in instance variables. See MediaRanker Revisited for an example of this.